docs/t-sql/statements/create-database-audit-specification-transact-sql.md

```diff
@@ -110,8 +112,39 @@ ADD (SELECT , INSERT
      ON HumanResources.EmployeePayHistory BY dbo )  
 WITH (STATE = ON) ;  
 GO  
-```  
+``` 
+
+### B. Audit any DML (INSERT, UPDATE or DELETE) on ALL tables in the sales schema for a specific database role  

```

Comment:
### B. Audit any DML (INSERT, UPDATE or DELETE) on _all_ tables in a schema for a specific database role

Reply:
Changed